apps/wallet/src/ui/app/pages/qredo-connect/components/QredoAccountItem.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { CheckFill16 } from '@mysten/icons';
+import { formatAddress } from '@mysten/sui.js';
+import cn from 'classnames';
+
+import { type Wallet } from '_src/shared/qredo-api';
+import { BadgeLabel } from '_src/ui/app/components/BadgeLabel';
+import { Text } from '_src/ui/app/shared/text';
+
+export type QredoAccountItemProps = Wallet & {
+    selected: boolean;
+    onClick: () => void;
+};
+
+export function QredoAccountItem({
+    selected,
+    address,
+    onClick,
+    labels,
+}: QredoAccountItemProps) {
+    return (
+        <div
+            className="flex flex-row items-center flex-nowrap group gap-3 py-4 cursor-pointer"
```

Comment:
nit: no need for `flex-row` since it's the default flex-direction

Reply:
Yeah usually I add it to make it explicit but yeah will remove it